package.json

```diff
@@ -77,7 +77,7 @@
     "start-prod": "tt-serve --config node_modules/terra-dev-site/config/webpack/webpack.config.js --env.disableHotReloading -p",
     "start-static": "npm run compile:prod && tt-serve-static --site ./build",
     "test": "npm run jest && npm run wdio",
-    "wdio": "wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
+    "wdio": "cross-env NODE_OPTIONS=--max-old-space-size=8192 wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
Out of curiosity, how much leeway does 8192?  Do we have any idea?

Reply:
lots, default is 512 mb this is 8 gb